### PR TITLE
reduce serializations

### PR DESCRIPTION
**Pull request by LefKok**
Making Sure we do not do unnecessary serializations when broadcasting messages

### CODE REVIEW

File: crates/sui-distributed-execution/src/exec_worker.rs [hunk 1]
```diff
@@ -279,99 +279,99 @@ impl<
...
-            );
-
-        let mut missing_objs = HashSet::new();
-        let input_object_map = input_objects.into_object_map();
-        for read_obj_id in &inner_temp_store.runtime_read_objects {
-            if !input_object_map.contains_key(read_obj_id) && !child_inputs.contains(read_obj_id) {
-                missing_objs.insert(*read_obj_id);
-            }
-        }
+    // async fn _async_exec(
```
Comment (asonnino) [inline crates/sui-distributed-execution/src/exec_worker.rs:282]: dead code

**Review by asonnino — APPROVED** [review]
Seems better indeed.
Only comment is to remove dead code (makes it hard to read)

File: crates/sui-distributed-execution/src/exec_worker.rs (hunk 1 above)
Comment (LefKok) [reply crates/sui-distributed-execution/src/exec_worker.rs:282]: There is a lot, but I don't know if we want it for other testing purposes. @zablotchi could you do a cleanup of whatever is unnecessary?

**Review by zablotchi — APPROVED** [review]
Looks good to me. Thanks!

**Pull request merged**